client/components/mobile/Selector.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
I think this file could have a more descriptive/specific name since "Selector" feels pretty generic, where this component is specifically being used for a Preference selection.

Reply:
True. I renamed it to `<PreferencePicker />` to be more descriptive